src/main/java/lotto/view/InputView.java

```diff
@@ -33,6 +42,46 @@ private static Lotto parseWinningNumbers(String input) {
     public static LottoNumber inputBonusNumber() {
         System.out.println("보너스 볼을 입력해 주세요.");
         return LottoNumber.of(Integer.parseInt(scanner.nextLine()));
+    }
+
+    public static int inputManualLottoCount() {
+        while (true) {
+            try {
+                System.out.println("\n수동으로 구매할 로또 수를 입력해 주세요.");
+                String input = scanner.nextLine();
+                validateNotEmpty(input);
+                return Integer.parseInt(input.trim());
+            } catch (IllegalArgumentException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+    }
+
+    public static ManualLottos inputManualLottos(int count) {
+        validateCount(count);
+        if (count == 0) {
+            return new ManualLottos(null);
+        }
 
+        System.out.println("수동으로 구매할 번호를 입력해 주세요.");
+        List<Lotto> manualLottos = new ArrayList<>();
```

Comment:
```suggestion
        List<String> manualLottos = new ArrayList<>();
```
앞에서 피드백한 바와 같이 String으로 저장하고 반환하는 것은 어떨까?